experiments/models.py

```diff
@@ -183,6 +185,16 @@ def __str__(self):
     session = models.ForeignKey(
         'ParticipantSession', on_delete=models.CASCADE, null=True)
 
+    @property
+    def status_label(self):
+        test = _('experiments:models:datapoint:label:test')
+        pilot = _('experiments:models:datapoint:label:pilot')
```

Comment:
I'd rather have these values in constants somewhere (Top of the file, top of the class, both could work)